packages/fx-core/src/common/samples.ts

```diff
@@ -68,11 +68,11 @@ class SampleProvider {
       this.branchOrTag = SampleConfigTag;
       const branch = process.env[FeatureFlagName.SampleConfigBranch];
       if (branch) {
-        const data = await this.fetchRawFileContent(branch);
-        if (data !== undefined) {
+        try {
+          const data = await this.fetchRawFileContent(branch);
           this.branchOrTag = branch;
           this.samplesConfig = data as { samples: Array<Record<string, unknown>> };
-        }
+        } catch (e: unknown) {}
```

Comment:
maybe record the telemetry data?

Reply:
This usually happens when user sets wrong env variable. So I think telemetry event is redundant.